designsafe/static/scripts/ng-designsafe/components/modals/data-browser-service-preview/data-browser-service-preview.component.js

```diff
@@ -53,6 +54,14 @@ class DataBrowserServicePreviewCtrl {
                                 const isGeoJson = validateGeoJson(body);
                                 if (isGeoJson) this.renderGeoJson(body);
                             }
+
+                            if (extension.includes('hazmapper')) {
```

Comment:
I think we want these `if` stanzas to be mutually exclusive, right? If so, we'll need an `else if` here instead of an `if`